Meta/build-native-partition.sh

```diff
@@ -16,7 +16,7 @@ cleanup() {
 
 if [ "$(id -u)" != 0 ]; then
     set +e
-    ${SUDO} -- sh -c "\"$0\" $* || exit 42"
+    ${SUDO} ${SHELL} -c -- "\"$0\" $* || exit 42"
```

Comment:
one more shellcheck complaint here:

```suggestion
    ${SUDO} "${SHELL}" -c -- "\"$0\" $* || exit 42"
```